spec/fixtures/mocks/incomplete_tasks_with_framework_and_active_submission.json

```diff
@@ -161,6 +161,39 @@
           }
         }
       }
+    },
+    {
+      "id": "b847e0f7-027e-4b95-afa2-3490b8d05a1d",
+      "type": "tasks",
+      "attributes": {
+        "status": "correcting",
+        "framework_id": "7a50a178-3fb8-4c0a-9f2c-8841812448d1",
+        "supplier_id": "533a1357-4faf-4ced-b759-3e6da0bc5f3e",
+        "supplier_name": "Bobs Cheese Shop",
+        "description": null,
+        "due_on": "2017-08-07",
+        "period_year": 2017,
+        "period_month": 7
+      },
+      "relationships": {
```

Comment:
We contemplated that, but that would massively balloon our fixtures, as really, all of them will have active and latest. I'm not sure the advantage of having more accurate fixtures is worth making them so massive, and making it harder to see the important bits. [james]

Reply:
I think "massively" might be over-stating it, especially as all but this task will have a "latest" that matches the "active", so it would only need to be added to the "relationships" section.

The advantage is that the specs are less brittle to changes to the application code and provide better coverage for the code that might check for the presence of active and latest.